community/rust/APKBUILD

```diff
@@ -10,7 +10,7 @@ _bootver=1.34.0
 pkgrel=0
 pkgdesc="The Rust Programming Language"
 url="https://www.rust-lang.org"
-arch="x86_64"
+arch="x86_64 armv7 armhf"
```

Comment:
You seem to have forgotten aarch64 and ppc64le here?

Reply:
No, I don't have tarballs for those arches for bootstrapping yet, I first need a arm* machine to test it on these two arches before I produce tarballs for ppc64le/aarch64. Thanks for looking at this though :)

Reply:
Added all arches now.